### DISCUSSION TITLE
How can I change the message: 'Started server http://localhost:3000'

### DISCUSSION THREAD

**Discussion by adrianbaena** [honojs/hono#2977] [Q&A] [answered]
I've been searching for the strings "Started" and "Started server" without any luck. Could it be possible to modify this string that prints every time the server is started? I'm using Bun btw.

Thanks for the help!

Adrián

**Comment by EdamAme-x** [comment]
This is a question regarding Bun and I don't think Hono is involved.

**Comment by NicoPlyley** [comment, marked as answer]
Hi @RotaN8 

The answer given by @EdamAme-x is correct. Bun is responsible for sending that message. Hono is only a router it does not handle the server being started or any lifecycles. 

To update the message you can use `Bun.serve(app)` and add a custom console log after it. 

If you do this, you do not need an export default anymore, when Bun sees a file has an export default it tries to run it in a Bun serve automatically.

**Reply by adrianbaena** [reply]
Thank you very much, @EdamAme-x and @NicoPlyley for your answer!

Should we add this to Bun's documentation inside Hono maybe? I think its simple but useful for some people that want to customize the default behavior.

**Reply by NicoPlyley** [reply]
@RotaN8 I think you're correct, it's mentioned a bit on using Bun serve, but I think showing how to customize it and use lifecycle methods would be a good addition. I will try to make a pr on this 

Please also mark a correct answer for any future people looking for this.

**Reply by adrianbaena** [reply]
That would be great! I think this thread can be closed then.

Thanks for your support.
